src/java.base/share/classes/java/security/interfaces/RSAPrivateKey.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1998, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This diff is redundant but no-op. You should merge openjdk/jdk's master branch to your PR branch, so the diff displayed by GitHub is up-to-date and this will go away.

Reply:
Sure, this comment serves as a note to reviewers that these 2 header changes have been committed in other changes and thus can be safely ignored.

Reply:
There is no harm in merging the upstream master, using `git merge`, to bring your branch more up-to-date. And it can be helpful in cases like this where it might help reviewers avoid extraneous information. It is also recommended in other cases (e.g., a long-running PR where there is a lot of drift between the PR source branch and the upstream target branch).

Skara will squash all commits anyway when integrating so the results will be identical.

Btw, one thing you should _not_ do is rebase a source branch of an active PR.